code/__defines/dcs/signals.dm

```diff
@@ -28,6 +28,10 @@
 /// fires on the target datum when an element is attached to it  (/datum/element)
 #define COMSIG_ELEMENT_DETACH_ON_HOST_DESTROY "ELEMENT_DETACH_ON_HOST_DESTROY"
 
+// /datum/record signals
+#define COMSIG_RECORD_CREATED "record_created"
+#define COMSIG_RECORD_MODIFIED "record_modified"
+
```

Comment:
They need to go in their own file under code\__defines\dcs\signals\signals_warrants.dm (or whatever you want to call it)